admin/templates/configuration/etc/init.d/mysql-proxy

```diff
@@ -70,7 +71,7 @@ if [ "$USER" = "root" ]; then
     proxy_user_option="--user=$proxy_user"
 fi
 
-cmd="DEBUG=${LUA_DEBUG_LEVEL} QSERV_RPC_PORT=${QSERV_RPC_PORT} ${DAEMON} \
+cmd="DEBUG=${LUA_DEBUG_LEVEL} QSERV_RPC_PORT=${QSERV_RPC_PORT} QSERV_CONFIG=${QSERV_CONFIG} ${DAEMON} \
```

Comment:
Indeed, we son't need QSERV_RPC_PORT anymore, will remove it.

Regarding environment - I don't know if there is any other way to pass configuration to Lua script. We could try to implement config file reader in Lua, but config file name itself needs to be passed somehow to Lua level.